CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ From v1.4, Orion requires Java 11. Orion on Java 8 is no longer supported.
 
 ## 1.5 RC 
 
+### Additions and Improvements 
+
+- Retrieve privacy group endpoint (/retrievePrivacyGroup) that will return the privacy group details and members for a given privacyGroupId [\#307](https://github.com/PegaSysEng/orion/pull/307)
```

Comment:
Reads better to me without the 'that' in 'that will return'

Reply:
agree - but not a big deal

Reply:
updated